datanode/gateway/graphql/resolvers.go

```diff
@@ -870,6 +870,135 @@ func (r *myQueryResolver) EstimateOrder(
 	}, nil
 }
 
+func (r *myQueryResolver) EstimateFees(
+	ctx context.Context,
+	market, party string,
+	price *string,
+	size string,
+	side vega.Side,
+	timeInForce vega.Order_TimeInForce,
+	expiration *int64,
+	ty vega.Order_Type,
+) (*FeeEstimate, error) {
+	order := &types.Order{}
+
+	var err error
+
+	// We need to convert strings to uint64 (JS doesn't yet support uint64)
+	if price != nil {
+		order.Price = *price
```

Comment:
Just wondering, because I noticed the price factor function a bit earlier, what decimal places will this price be in? Seeing as this is part of the user-facing API, are we using asset decimals, and later converting to market precision? 

TL;DR: are we taking decimals in to account here?

Reply:
It does the conversion of price and volume / order size, but would be good if you could have a closer look at the method and check that it's doing it correctly.

Reply:
My concern is that the price the datanode ingests is using market precision (e.g. asset being ETH - 18 decimals, market using 5 decimals => if the price on the event received by the datanode is 1, that equates to 0.00001 ETH, not 1 wei). I'll take a look though